src/recipes/burnt-caramel-manhattan.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "Burnt Caramel Manhattan",
+    "description": "The Burnt Caramel Manhattan is a sweet, decadent variation on the classic Manhattan.",
+    "github": "kristina-hager",
+    "ingredients": [
+        {
+            "quantity": "2",
+            "measure": "oz",
+            "ingredient": "bourbon"
+        },
+        {
+            "quantity": ".25",
+            "measure": "oz",
+            "ingredient": "sweet vermouth"
+        },
+        {
+            "quantity": ".25",
+            "measure": "oz",
+            "ingredient": "buttershot schnapps"
+        },
+        {
+            "quantity": ".5",
+            "measure": "oz",
+            "ingredient": "raspberry Chambord"
+        },
+        {
+            "quantity": "3",
+            "measure": "dashes",
+            "ingredient": "angostura bitters"
+        }
+    ],
+    "directions": [
+        "Chill a cocktail glass with ice and soda.",
+        "Fill a mixing glass with ice and then add all ingredients.",
+        "Using a bar spoon, stir the drink for approximately 20 seconds.",
+        "When stirring, always ensure the glass is topped up with ice.",
+        "Taste and then strain into the chilled cocktail glass, serving straight up.",
+        "Note: You can vary this by increasing buttershot schnapps and vermouth to .5 oz."
+    ],
+    "image": "manhattan.jpg",
```

Comment:
Hi, can you add an image specific to `burnt-caramel-manhattan.jpg`?

Reply:
well, the drinks appear identical. however, i can take a photo at home, though it will need to wait until an appropriate cocktail hour tonight :)

Reply:
Ok, sounds good. 👍

Reply:
updated with photo